public/main.js

```diff
@@ -65,34 +79,33 @@ function reloadTodoList() {
             listItem.textContent = todo.title;
             var delButton = document.createElement("button");
             delButton.id = "del" + numTodos;
-            delButton.className = "deleteButton";
             delButton.textContent = "delete";
             delButton.onclick = function (event) {
-                deleteTodo(todo.id, function() {
+                deleteTodo(todo.id, function () {
```

Comment:
One last thing, it looks like the deleteButton creation and completeButton creation is basically the same, can they use a common function?